julia-mode.el

```diff
@@ -787,6 +787,15 @@ Return nil if point is not in a function, otherwise point."
   (set (make-local-variable 'indent-line-function) 'julia-indent-line)
   (set (make-local-variable 'beginning-of-defun-function) #'julia-beginning-of-defun)
   (set (make-local-variable 'end-of-defun-function) #'julia-end-of-defun)
+  ;; If completion before point has higher priority than around, \lamb
+  ;; can get completed to \lambdamb
+  (add-hook 'completion-at-point-functions #'julia-mode-latexsub-completion-at-point-before)
+  (add-hook 'completion-at-point-functions #'julia-mode-latexsub-completion-at-point-around)
+  (when julia-force-tab-complete
+    (setq-local tab-always-indent 'complete)
```

Comment:
The macro `setq-local` was added in emacs 24.3. I'm assuming we're only supporting the latest minor version of all the major versions of emacs we support, but if not, this needs to be changed to `(set (make-local-variable 'tab-always-indent) 'complete)`. I can also add a commit changing all usage of `(set (make-local variable ...) ...)` to `setq-local` for stylistic consistency if desired.

Reply:
I am fine with this. Let's keep things simple and go with 24.3 as in #101.

Reply:
One issue I've noticed, is now my current `julia-mode-abbrev-table` elements don't expand anymore due to the addition of the specialized prefix.  Perhaps the latex table could have a non-default name?

Reply:
I think the `julia-mode-abbrev-table` symbol rightfully belongs to this package, and you should expect to sometimes have symbols overridden if you put them in a package's namespace.

Reply:
I does, but you've given it a prefix that won't let any normal abbrevs work